--- libva-intel-driver-1.0.19/configure.ac.orig	2012-11-09 02:44:47.000000000 +0100
+++ libva-intel-driver-1.0.19/configure.ac	2013-02-17 11:26:57.649047570 +0100
@@ -22,7 +22,7 @@
 AC_CONFIG_SRCDIR([Makefile.am])
 AM_INIT_AUTOMAKE([1.9 tar-ustar])
 
-AM_CONFIG_HEADER([src/config.h])
+AC_CONFIG_HEADERS([src/config.h])
 
 INTEL_DRIVER_MAJOR_VERSION=intel_driver_major_version
 INTEL_DRIVER_MINOR_VERSION=intel_driver_minor_version
